README.md

```diff
@@ -20,9 +20,53 @@ This chart deploys a fully functioning instance of the [Fn](https://github.com/f
 
 
 ```bash
-helm init
+helm init --upgrade
 ```
 
+## Preparing chart values
+
+### Minimum configuration
+
+In order to get a working deployment please pay attention to what you have in your chart values.
+[Here](fn/values.yaml) you may find bare minimum chart values configuration necessary to deploy a working Fn cluster.
+
+### Exposing Fn services
+
+#### Ingress controller
+
+If you are installing Fn behind an ingress controller you'd need to have only 1 DNS sub-domain that will stand for an ingress controller IP resolve.
+
+Please pay attention, an ingress controller works as a proxy, it means that you can use ingress IP address as an HTTP proxy:
+
+```
+curl -x http://<ingress-controller-endpoint>:80 api.fn.internal 
+{"goto":"https://github.com/fnproject/fn","hello":"world!"}
+```
+
+
+#### LoadBalancer
+
+In order to use native way to expose Fn services you'd need to modify Fn API, Fn runner LB and UI service definitions:
```

Comment:
In order to natively expose the Fn services, you'll need to modify the Fn API, Runner, and UI service definitions: